src/pages/FMTeam/main.ts

```diff
@@ -77,12 +65,33 @@ export const FMTeam: pageInterface = {
     api.storage.addStyle(
       require('!to-string-loader!css-loader!less-loader!./style.less').toString(),
     );
-    j.$(document).ready(function () {
+    j.$(document).ready(() => {
+      utils.changeDetect(
+        () => {
+          page.reset();
+          start();
+        },
+        () => {
+          return j.$('head > meta[property="og:title"]').text() || '';
+        },
+      );
+    });
+    function start() {
       const urlSegment = page.url.split('/')[3];
-      const handlingPage = urlSegment === 'read' || urlSegment === 'series';
+      const handlingPage = urlSegment === 'read' || urlSegment === 'comics';
+
       if (handlingPage && typeof page.url.split('/')[4] !== 'undefined') {
-        page.handlePage();
+        con.info('Waiting');
+        utils.waitUntilTrue(
```

Comment:
Please reset the waitUntilTrue to make sure it does not queue multiple executions.
https://github.com/MALSync/MALSync/blob/9a02c217ee6722d29a5d679d0b9738177c5e013a/src/pages/Puray/main.ts#L50